activejob/lib/active_job/test_helper.rb

```diff
@@ -239,6 +241,14 @@ def enqueued_jobs_size(only: nil)
             enqueued_jobs.size
           end
         end
+
+        def serialize_args_for_assertion(args)
+          original_args = args.dup
+          if job_args = args.delete(:args)
+            args[:args] = ActiveJob::Arguments.serialize(job_args)
+          end
+          original_args
```

Comment:
This is weird. Why `original_args` is returned and what is mutated is `args`? Is not better to mutate the `args.dup` and return it?